### PR TITLE
Separate Hyundai Ioniq Hybrid and EV 2020 Premium SE #1969

### PR DESCRIPTION
**Pull request by baldwalker**
resubmitted PR due to original branch getting messed up

### CODE REVIEW

**Comment by adeebshihadeh** [comment]
Looks good, we'll just need a test route from this branch added to test_car_models.py

File: selfdrive/car/hyundai/values.py [hunk 1]
```diff
@@ -23,8 +23,9 @@ class CAR:
   ELANTRA = "HYUNDAI ELANTRA LIMITED ULTIMATE 2017"
   ELANTRA_GT_I30 = "HYUNDAI I30 N LINE 2019 & GT 2018 DCT"
   HYUNDAI_GENESIS = "HYUNDAI GENESIS 2015-2016"
-  IONIQ = "HYUNDAI IONIQ ELECTRIC PREMIUM SE 2020"
+  IONIQ = "HYUNDAI IONIQ HYBRID 2017-2019"
   IONIQ_EV_LTD = "HYUNDAI IONIQ ELECTRIC LIMITED 2019"
+  IONIQ_EV_2020 = "HYUNDAI IONIQ ELECTRIC ALL MODELS 2020"
```
Comment (adeebshihadeh) [inline selfdrive/car/hyundai/values.py:28]: remove "all models"

**Comment by baldwalker** [comment]
all done

**Comment by baldwalker** [comment]
> Looks good, we'll just need a test route from this branch added to test_car_models.py

all done

**Pull request merged**